docs/index.md

```diff
@@ -24,38 +24,24 @@ See the relationship between the connector files (in blue) and the Tableau **Con
 
 ![]({{ site.baseurl }}/assets/files-overview.png)
 
-## Before you begin
+# What is a Taco?
+A `.taco` file is a packaged Tableau connector file that can be dropped into your `My Tableau Repository/Connectors` folder. They will be automatically loaded by Tableau.
 
-You need to do these things before you start:
-
-- Install the ODBC or JDBC driver that you’ll use with the connector you’ll create.
-- Install Tableau Desktop 2019.2 or later on a Windows or Mac computer.
-
-## Using a Connector
+For more information about packaging your connector into a Taco, refer to [How to Package and Sign Your Connector for Distribution]({{ site.baseurl }}/docs/package-sign)
 
-You must start Tableau Desktop or Tableau Server with a special command line argument that tells Tableau where to find your Connector. See [Run Your Connector]({{ site.baseurl }}/docs/share) for more information.
+# Before you Begin
 
-**Tableau Desktop:** 
-
-For Windows:
-1. Create a directory for Tableau connectors. For example: `C:\tableau_connectors`
-1. Put the folder containing your connector's manifest.xml file in this directory. Each connector should have its own folder. For example: `C:\tableau_connectors\my_connector`
-1. Run Tableau using the `-DConnectPluginsPath` command line argument, pointing to your connector directory. For example: 
-
-    ```
-    tableau.exe -DConnectPluginsPath=C:\tableau_connectors
-    ```
+You need to do these things before you start:
 
-For macOS:
+- Install the ODBC or JDBC driver that you’ll use with the connector you’ll create.
+- Install Tableau Desktop 2019.2 or later on a Windows or Mac computer. Tableau Desktop 2019.4 or is required to work with taco files.
 
-In the following examples, replace [user name] with your name (for example /Users/agarcia/tableau_connectors) and [Tableau version] with the version of Tableau that you’re running (for example, 2019.3.app).
+# Using a Connector
 
-1. Create a directory for Tableau connectors. For example: `/Users/[user name]/tableau_connectors`
-1. Put the folder containing your connector's manifest.xml file in this directory. Each connector should have its own folder. For example: `/Users/[user name]/tableau_connector/my connector`
-1. Run Tableau using the `-DConnectPluginsPath` command line argument, pointing to your connector directory. For example: 
+## Packaged Connector (Taco)
+Simply drop your packaged `.taco` file into your `My Tableau Repository/Connectors` and launch Tableau. See [How to Run your Packaged Connector (Taco)]({{ site.baseurl }}/docs/share) for more information.
```

Comment:
Remove "Simply".  Drop your packaged...    Remove "How to" from the "Run Your Packaged Connector (.taco)".